src/components/DiagramFrame/DiagramFrame.tsx

```diff
@@ -127,23 +118,12 @@ export const DiagramFrame: React.FC<DiagramFrameProps> = ({
     <Layout hasAside height="100%">
       <Rail width="50px">
         <SpaceVertical style={{alignItems: "center"}} alignItems="center" gap="xsmall">
-          <IconButton
-            icon={<Dashboard />}
-            label="Diagram"
-            tooltipPlacement="right"
-            size="large"
-            toggle={onlyDiagramSelected()}
-            onClick={showDiagramFullPage}
-            style={{color: onlyDiagramSelected() && OVERRIDE_KEY, 
-              backgroundColor: onlyDiagramSelected() && OVERRIDE_KEY_SUBTLE,
-              borderRadius: "10px"}}
-          />
           <IconButton
             icon={<AccountTree />}
             label="Settings"
             tooltipPlacement="right"
             size="large"
-            onClick={() => isDiagramFullPage() && setShowSettings(!showSettings)}
+            onClick={() => {closePanels(); setShowSettings(!showSettings)}}
```

Comment:
consider creating a toggleShowSettings method. It would call closePanels and setShowSettings. I think it reads cleaner than this inline function.